common/combat_tactics/00_combat_tactics.txt

```diff
@@ -1761,36 +1912,46 @@ charging_own_skirmishers_tactic = {
 			leader = {
 				NOT = { martial = 10 }
 			}
-			OR = {
-				light_troops = {
-					who = light_cavalry
-					value = 0.1
-				}
-				light_troops = {
-					who = camel_cavalry
-					value = 0.1
-				}
-			}
+			light_cavalry_troops_010_tactic_trigger = yes
 		}
 	}
 
 	change_phase_to = melee
-
-	knights_offensive = 3
+	
+	# horse_archers
+	horse_archers_defensive = -0.3
+	horse_archers_offensive = -1.2
+	
+	# light_infantry # get modifiers of archers
+	light_infantry_defensive = -0.5
+	archmage_defensive = -0.5
+	light_infantry_offensive = -4.2
+	archmage_offensive = -4.2
+	
+	# pikemen # replaces vanilla knights
+	pikemen_offensive = 3
+	war_machines_offensive = 3
+	
+	# light_cavalry
 	light_cavalry_offensive = 1.8
-	camel_cavalry_offensive = 1.8
+	animals_offensive = 1.8
 	
-	archers_offensive = -4.2
+	# knights # replaces vanilla knights
+	knights_offensive = 3
+	dragons_offensive = 3
+	
+	# archers
 	archers_defensive = -0.5
-	horse_archers_offensive = -1.2
-	horse_archers_defensive = -0.3
+	# siege_machines_defensive = -0.5	# In back of army
+	archers_offensive = -4.2
+	# siege_machines_offensive = -4.2	# In back of army
```

Comment:
> In back of army

Just like archers and mages?
Why siege machines don't get offensive debuff? Or why any other melee troops don't get defensive debuff?
You can't shoot things in melee battle direction and hope that your melee troops will remain unharmed.